README.md

```diff
@@ -141,7 +141,7 @@ TBD
 
 See [CONTRIBUTING.md](CONTRIBUTING.md)
 
-We meet weekly Monday's at 8:15 PT. The meeting is subject to change depending on
+We meet weekly Monday's at 8:15 AM PT. The meeting is subject to change depending on
```

Comment:
I don't have strong opinion for this change, I think it is following https://github.com/open-telemetry/community#special-interest-groups.

Reply:
mm.. the client instrument has AM specified!